snaps_k8s/provision/ansible_configuration.py

```diff
@@ -902,3 +929,189 @@ def delete_weave_interface(k8s_conf):
         }
         ansible_utils.apply_playbook(consts.K8_DELETE_WEAVE_INTERFACE,
                                      variables=pb_vars)
+def __get_loadbalancer_dict(config):
+    """
+    function to get loadbalancer dictionary
+    :param config:
+    :return:
+    """
+    logger.info("Argument List:\n config: %s", config)
+    for item in config_utils.get_ha_config(config):
+        return item.get(consts.HA_API_EXT_LB_KEY)
+                        
+
+def __get_hostname_map(hosts):
```

Comment:
Please leverage config_utils#get_hostname_ips_dict() or get_nodes_ip_name_type() and remove this function.

Reply:
I have implemented in the 'ha_ci_scripts' branch

Reply:
implemented in 'ha_ci_scripts' branch